js/mock.ts

```diff
@@ -0,0 +1,53 @@
+import { getRandomInteger, getRandomId } from './utils';
+import {DESCRIPTION_TEXTS, MESSAGES, NAMES} from './constants';
+import {Photo, PhotoComment} from './types';
+
+const enum Default {
+	ARRAY_LENGTH = 25,
+	LENGTH_MIN = 0,
+	LENGTH_MAX = 30
+}
+
+const enum Img {
+	ID_MIN = 1,
+	ID_MAX = 25,
+}
+
+const enum ImgComment {
+	ID_MIN = 1,
+	ID_MAX = 1000,
+}
+
+const enum Avatar {
+	URL_MIN = 1,
+	URL_MAX = 6,
+}
+
+const getRandomImgUrl = getRandomId(Img.ID_MIN, Img.ID_MAX);
+const getRandomImgId = getRandomId(Img.ID_MIN, Img.ID_MAX);
+const getCommentId = getRandomId(ImgComment.ID_MIN, ImgComment.ID_MAX);
+const getAvatarUrl = getRandomId(Avatar.URL_MIN, Avatar.URL_MAX);
```

Comment:
Немного странная реализация `getRandomId` которая позволяет переполнять отрезок и создавать заново. Проще сделать функции, которые возвращают случайное число или как тут шаблонную строку 
```suggestion
const getAvatarUrl = () => `img/avatar-${getRandomInteger(Avatar.URL_MIN, Avatar.URL_MAX)}.svg` as const;
```